airflow/task/task_runner/base_task_runner.py

```diff
@@ -86,6 +87,10 @@ def __init__(self, local_task_job):
 
         # pylint: disable=consider-using-with
         self._error_file = NamedTemporaryFile(delete=True)
+
+        # HOTFIX: When reporting exceptions, this file was usually locked because it was still opened by this process
+        self._error_file.close()
```

Comment:
This looks wrong. [The documentation says this would destroy the file immediately](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile), rendering this attribute useless.

Reply:
On Windows we basically just create an empty text file. Not closing it immediately raised an exception while logging since the file was opened by another python process.

Reply:
Ok nevermind... I removed the line and it still works on our Dask-Worker.